x-pack/metricbeat/module/oracle/connection.go

```diff
@@ -0,0 +1,84 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package oracle
+
+import (
+	"database/sql"
+
+	"gopkg.in/goracle.v2"
+
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+
+	// Register driver.
+	_ "gopkg.in/goracle.v2"
+
+	"github.com/pkg/errors"
```

Comment:
Nit. Move it up with `gopkg.in/goracle.v2`.

Reply:
Done! :slightly_smiling_face: